pipelines/upload_transacao_cct/tasks.py

```
@@ -0,0 +1,277 @@
+                AND "{data_ordem_end}"
+            """
+
+    transacao_select = f"""
+        SELECT
+            {{cols}}
+        FROM
+            {project_id}.{constants.TRANSACAO_CCT_VIEW_NAME.value}
+        WHERE
+            {export_filter}
+        LIMIT {{count}}
+    """
```

Ruff (0.14.0)

Possible SQL injection vector through string-based query construction

(S608)